commit a222e812eb0e3528633d8d2735d7cfea3f6891b3
Author: Sylvestre Ledru <sylvestre@debian.org>
Date:   Fri Oct 11 23:54:42 2019 +0200

    Revert "[clang-tools-extra] [cmake] Link against libclang-cpp whenever possible"
    
    This reverts commit 0820041e1daa158f39ab2e414d03c7eb2d1260d7.

Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-apply-replacements/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-apply-replacements/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-apply-replacements/tool/CMakeLists.txt
@@ -5,15 +5,12 @@ set(LLVM_LINK_COMPONENTS
 add_clang_tool(clang-apply-replacements
   ClangApplyReplacementsMain.cpp
   )
-clang_target_link_libraries(clang-apply-replacements
+target_link_libraries(clang-apply-replacements
   PRIVATE
+  clangApplyReplacements
   clangBasic
   clangFormat
   clangRewrite
   clangToolingCore
   clangToolingRefactoring
   )
-target_link_libraries(clang-apply-replacements
-  PRIVATE
-  clangApplyReplacements
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-change-namespace/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-change-namespace/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-change-namespace/tool/CMakeLists.txt
@@ -7,11 +7,12 @@ set(LLVM_LINK_COMPONENTS
 add_clang_tool(clang-change-namespace
   ClangChangeNamespace.cpp
   )
-clang_target_link_libraries(clang-change-namespace
+target_link_libraries(clang-change-namespace
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
+  clangChangeNamespace
   clangFormat
   clangFrontend
   clangRewrite
@@ -19,7 +20,3 @@ clang_target_link_libraries(clang-change
   clangTooling
   clangToolingCore
   )
-target_link_libraries(clang-change-namespace
-  PRIVATE
-  clangChangeNamespace
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-doc/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-doc/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-doc/tool/CMakeLists.txt
@@ -4,19 +4,16 @@ add_clang_tool(clang-doc
   ClangDocMain.cpp
   )
 
-clang_target_link_libraries(clang-doc
+target_link_libraries(clang-doc
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
   clangFrontend
+  clangDoc
   clangTooling
   clangToolingCore
   )
-target_link_libraries(clang-doc
-  PRIVATE
-  clangDoc
-  )
 
 install(FILES ../assets/clang-doc-default-stylesheet.css
   DESTINATION share/clang
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-include-fixer/find-all-symbols/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-include-fixer/find-all-symbols/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-include-fixer/find-all-symbols/tool/CMakeLists.txt
@@ -4,7 +4,7 @@ add_clang_tool(find-all-symbols
   FindAllSymbolsMain.cpp
   )
 
-clang_target_link_libraries(find-all-symbols
+target_link_libraries(find-all-symbols
   PRIVATE
   clangAST
   clangASTMatchers
@@ -13,9 +13,6 @@ clang_target_link_libraries(find-all-sym
   clangLex
   clangSerialization
   clangTooling
-  )
-target_link_libraries(find-all-symbols
-  PRIVATE
   findAllSymbols
   )
 
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-include-fixer/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-include-fixer/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-include-fixer/tool/CMakeLists.txt
@@ -4,19 +4,16 @@ add_clang_tool(clang-include-fixer
   ClangIncludeFixer.cpp
   )
 
-clang_target_link_libraries(clang-include-fixer
+target_link_libraries(clang-include-fixer
   PRIVATE
   clangBasic
   clangFormat
   clangFrontend
+  clangIncludeFixer
   clangRewrite
   clangSerialization
   clangTooling
   clangToolingCore
-  )
-target_link_libraries(clang-include-fixer
-  PRIVATE
-  clangIncludeFixer
   findAllSymbols
   )
 
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-move/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-move/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-move/tool/CMakeLists.txt
@@ -4,19 +4,16 @@ add_clang_tool(clang-move
   ClangMove.cpp
   )
 
-clang_target_link_libraries(clang-move
+target_link_libraries(clang-move
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
   clangFormat
   clangFrontend
+  clangMove
   clangRewrite
   clangSerialization
   clangTooling
   clangToolingCore
   )
-target_link_libraries(clang-move
-  PRIVATE
-  clangMove
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-query/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-query/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-query/tool/CMakeLists.txt
@@ -3,17 +3,14 @@ include_directories(${CMAKE_CURRENT_SOUR
 add_clang_tool(clang-query
   ClangQuery.cpp
   )
-clang_target_link_libraries(clang-query
+target_link_libraries(clang-query
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
   clangDynamicASTMatchers
   clangFrontend
+  clangQuery
   clangSerialization
   clangTooling
   )
-target_link_libraries(clang-query
-  PRIVATE
-  clangQuery
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-reorder-fields/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-reorder-fields/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-reorder-fields/tool/CMakeLists.txt
@@ -2,16 +2,13 @@ add_clang_tool(clang-reorder-fields
   ClangReorderFields.cpp
   )
 
-clang_target_link_libraries(clang-reorder-fields
+target_link_libraries(clang-reorder-fields
   PRIVATE
   clangBasic
   clangFrontend
+  clangReorderFields
   clangRewrite
   clangSerialization
   clangTooling
   clangToolingCore
   )
-target_link_libraries(clang-reorder-fields
-  PRIVATE
-  clangReorderFields
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-tidy/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-tidy/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-tidy/CMakeLists.txt
@@ -31,7 +31,7 @@ add_clang_library(clangTidy
   )
 
 if(CLANG_ENABLE_STATIC_ANALYZER)
-  clang_target_link_libraries(clangTidy PRIVATE
+  target_link_libraries(clangTidy PRIVATE
     clangStaticAnalyzerCore
     clangStaticAnalyzerFrontend
   )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-tidy/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clang-tidy/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clang-tidy/tool/CMakeLists.txt
@@ -11,17 +11,14 @@ add_clang_tool(clang-tidy
 add_dependencies(clang-tidy
   clang-resource-headers
   )
-clang_target_link_libraries(clang-tidy
+target_link_libraries(clang-tidy
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
+  clangTidy
   clangTooling
   clangToolingCore
-  )
-target_link_libraries(clang-tidy
-  PRIVATE
-  clangTidy
   ${ALL_CLANG_TIDY_CHECKS}
   )
 
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/fuzzer/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clangd/fuzzer/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/fuzzer/CMakeLists.txt
@@ -12,16 +12,13 @@ add_llvm_fuzzer(clangd-fuzzer
   DUMMY_MAIN DummyClangdMain.cpp
   )
 
-clang_target_link_libraries(clangd-fuzzer
+target_link_libraries(clangd-fuzzer
   PRIVATE
   clangBasic
+  clangDaemon
   clangFormat
   clangFrontend
   clangSema
   clangTooling
   clangToolingCore
   )
-target_link_libraries(clangd-fuzzer
-  PRIVATE
-  clangDaemon
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/index/dex/dexp/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clangd/index/dex/dexp/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/index/dex/dexp/CMakeLists.txt
@@ -9,11 +9,8 @@ add_clang_executable(dexp
   Dexp.cpp
   )
 
-clang_target_link_libraries(dexp
-  PRIVATE
-  clangBasic
-  )
 target_link_libraries(dexp
   PRIVATE
+  clangBasic
   clangDaemon
   )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/indexer/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clangd/indexer/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/indexer/CMakeLists.txt
@@ -8,16 +8,13 @@ add_clang_executable(clangd-indexer
   IndexerMain.cpp
   )
 
-clang_target_link_libraries(clangd-indexer
+target_link_libraries(clangd-indexer
   PRIVATE
   clangAST
   clangBasic
+  clangDaemon
   clangFrontend
   clangIndex
   clangLex
   clangTooling
 )
-target_link_libraries(clangd-indexer
-  PRIVATE
-  clangDaemon
-)
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/tool/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clangd/tool/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/tool/CMakeLists.txt
@@ -15,10 +15,12 @@ if(CLANGD_BUILD_XPC)
   list(APPEND CLANGD_XPC_LIBS "clangdXpcJsonConversions" "clangdXpcTransport")
 endif()
 
-clang_target_link_libraries(clangd
+target_link_libraries(clangd
   PRIVATE
   clangAST
   clangBasic
+  clangTidy
+  clangDaemon
   clangFormat
   clangFrontend
   clangSema
@@ -26,10 +28,5 @@ clang_target_link_libraries(clangd
   clangToolingCore
   clangToolingRefactoring
   clangToolingSyntax
-  )
-target_link_libraries(clangd
-  PRIVATE
-  clangTidy
-  clangDaemon
   ${CLANGD_XPC_LIBS}
   )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/unittests/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clangd/unittests/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/unittests/CMakeLists.txt
@@ -78,26 +78,23 @@ add_unittest(ClangdUnitTests ClangdTests
   $<TARGET_OBJECTS:obj.clangDaemonTweaks>
   )
 
-clang_target_link_libraries(ClangdTests
+target_link_libraries(ClangdTests
   PRIVATE
   clangAST
   clangBasic
+  clangDaemon
   clangFormat
   clangFrontend
   clangIndex
   clangLex
   clangSema
   clangSerialization
+  clangTidy
   clangTooling
   clangToolingCore
   clangToolingInclusions
   clangToolingRefactoring
   clangToolingSyntax
-  )
-target_link_libraries(ClangdTests
-  PRIVATE
-  clangDaemon
-  clangTidy
   LLVMSupport
   LLVMTestingSupport
   )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/xpc/test-client/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/clangd/xpc/test-client/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/clangd/xpc/test-client/CMakeLists.txt
@@ -13,17 +13,14 @@ set(LLVM_LINK_COMPONENTS
     support
 )
 
-clang_target_link_libraries(clangd-xpc-test-client
+target_link_libraries(clangd-xpc-test-client
   PRIVATE
   clangBasic
+  clangDaemon
   clangFormat
   clangFrontend
   clangSema
   clangTooling
   clangToolingCore
-)
-target_link_libraries(clangd-xpc-test-client
-  PRIVATE
-  clangDaemon
   clangdXpcJsonConversions
 )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/modularize/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/modularize/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/modularize/CMakeLists.txt
@@ -11,7 +11,7 @@ add_clang_tool(modularize
   PreprocessorTracker.cpp
   )
 
-clang_target_link_libraries(modularize
+target_link_libraries(modularize
   PRIVATE
   clangAST
   clangBasic
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/pp-trace/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/pp-trace/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/pp-trace/CMakeLists.txt
@@ -7,7 +7,7 @@ add_clang_tool(pp-trace
   PPCallbacksTracker.cpp
   )
 
-clang_target_link_libraries(pp-trace
+target_link_libraries(pp-trace
   PRIVATE
   clangAST
   clangBasic
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/tool-template/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/tool-template/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/tool-template/CMakeLists.txt
@@ -6,7 +6,7 @@ add_clang_executable(tool-template
   ToolTemplate.cpp
   )
 
-clang_target_link_libraries(tool-template
+target_link_libraries(tool-template
   PRIVATE
   clangAST
   clangASTMatchers
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-apply-replacements/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/unittests/clang-apply-replacements/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-apply-replacements/CMakeLists.txt
@@ -11,13 +11,10 @@ add_extra_unittest(ClangApplyReplacement
   ApplyReplacementsTest.cpp
   )
 
-clang_target_link_libraries(ClangApplyReplacementsTests
+target_link_libraries(ClangApplyReplacementsTests
   PRIVATE
+  clangApplyReplacements
   clangBasic
   clangToolingCore
   clangToolingRefactoring
   )
-target_link_libraries(ClangApplyReplacementsTests
-  PRIVATE
-  clangApplyReplacements
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-change-namespace/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/unittests/clang-change-namespace/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-change-namespace/CMakeLists.txt
@@ -15,11 +15,12 @@ add_extra_unittest(ClangChangeNamespaceT
   ChangeNamespaceTests.cpp
   )
 
-clang_target_link_libraries(ClangChangeNamespaceTests
+target_link_libraries(ClangChangeNamespaceTests
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
+  clangChangeNamespace
   clangFormat
   clangFrontend
   clangRewrite
@@ -27,7 +28,3 @@ clang_target_link_libraries(ClangChangeN
   clangTooling
   clangToolingCore
   )
-target_link_libraries(ClangChangeNamespaceTests
-  PRIVATE
-  clangChangeNamespace
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-doc/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/unittests/clang-doc/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-doc/CMakeLists.txt
@@ -20,11 +20,12 @@ add_extra_unittest(ClangDocTests
   YAMLGeneratorTest.cpp
   )
 
-clang_target_link_libraries(ClangDocTests
+target_link_libraries(ClangDocTests
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
+  clangDoc
   clangFormat
   clangFrontend
   clangRewrite
@@ -32,7 +33,3 @@ clang_target_link_libraries(ClangDocTest
   clangTooling
   clangToolingCore
   )
-target_link_libraries(ClangDocTests
-  PRIVATE
-  clangDoc
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-include-fixer/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/unittests/clang-include-fixer/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-include-fixer/CMakeLists.txt
@@ -16,19 +16,16 @@ add_extra_unittest(ClangIncludeFixerTest
   FuzzySymbolIndexTests.cpp
   )
 
-clang_target_link_libraries(ClangIncludeFixerTests
+target_link_libraries(ClangIncludeFixerTests
   PRIVATE
   clangBasic
   clangFormat
   clangFrontend
+  clangIncludeFixer
   clangRewrite
   clangSerialization
   clangTooling
   clangToolingCore
-  )
-target_link_libraries(ClangIncludeFixerTests
-  PRIVATE
-  clangIncludeFixer
   findAllSymbols
   )
 
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-include-fixer/find-all-symbols/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/unittests/clang-include-fixer/find-all-symbols/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-include-fixer/find-all-symbols/CMakeLists.txt
@@ -12,7 +12,7 @@ add_extra_unittest(FindAllSymbolsTests
   FindAllSymbolsTests.cpp
   )
 
-clang_target_link_libraries(FindAllSymbolsTests
+target_link_libraries(FindAllSymbolsTests
   PRIVATE
   clangAST
   clangASTMatchers
@@ -21,8 +21,5 @@ clang_target_link_libraries(FindAllSymbo
   clangLex
   clangSerialization
   clangTooling
-  )
-target_link_libraries(FindAllSymbolsTests
-  PRIVATE
   findAllSymbols
   )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-move/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/unittests/clang-move/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-move/CMakeLists.txt
@@ -15,19 +15,16 @@ add_extra_unittest(ClangMoveTests
   ClangMoveTests.cpp
   )
 
-clang_target_link_libraries(ClangMoveTests
+target_link_libraries(ClangMoveTests
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
   clangFormat
   clangFrontend
+  clangMove
   clangRewrite
   clangSerialization
   clangTooling
   clangToolingCore
   )
-target_link_libraries(ClangMoveTests
-  PRIVATE
-  clangMove
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-query/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/unittests/clang-query/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-query/CMakeLists.txt
@@ -11,17 +11,14 @@ add_extra_unittest(ClangQueryTests
   QueryParserTest.cpp
   )
 
-clang_target_link_libraries(ClangQueryTests
+target_link_libraries(ClangQueryTests
   PRIVATE
   clangAST
   clangASTMatchers
   clangBasic
   clangDynamicASTMatchers
   clangFrontend
+  clangQuery
   clangSerialization
   clangTooling
   )
-target_link_libraries(ClangQueryTests
-  PRIVATE
-  clangQuery
-  )
Index: llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-tidy/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_10~svn374603.orig/clang-tools-extra/unittests/clang-tidy/CMakeLists.txt
+++ llvm-toolchain-snapshot_10~svn374603/clang-tools-extra/unittests/clang-tidy/CMakeLists.txt
@@ -21,7 +21,7 @@ add_extra_unittest(ClangTidyTests
   TransformerClangTidyCheckTest.cpp
   )
 
-clang_target_link_libraries(ClangTidyTests
+target_link_libraries(ClangTidyTests
   PRIVATE
   clangAST
   clangASTMatchers
@@ -32,9 +32,6 @@ clang_target_link_libraries(ClangTidyTes
   clangTooling
   clangToolingCore
   clangTransformer
-  )
-target_link_libraries(ClangTidyTests
-  PRIVATE
   clangTidy
   clangTidyAndroidModule
   clangTidyGoogleModule
@@ -42,4 +39,5 @@ target_link_libraries(ClangTidyTests
   clangTidyObjCModule
   clangTidyReadabilityModule
   clangTidyUtils
+  clangToolingRefactoring
   )
